openslides_backend/action/actions/vote/create.py

```diff
@@ -43,15 +43,12 @@ def prefetch(self, action_data: ActionData) -> None:
             use_changed_models=False,
         )
         fields = [
-            "vote_$_ids",
```

Comment:
Shouldn't we use instead the new fields? But I don't know, what they are for!

Reply:
Add the new fields here.